app/api/dao/admin.py

```diff
@@ -87,3 +87,19 @@ def revoke_admin_user(user_id, data):
             return messages.USER_ADMIN_STATUS_WAS_REVOKED, 200
 
         return messages.USER_DOES_NOT_EXIST, 404
+
+    @staticmethod
+    def list_admins(user_id):
+        """ Retrieves a list of admin users for the user with specified ID.
+        
+        Arguments:
+            user_id: The ID of the user querying the fellow admins.
+        
+        Returns:
+            A list of admin users matching conditions and the HTTP response code.
+        """
+
+        users_list = UserModel.query.filter(UserModel.id != user_id).all()
+        list_of_users = [user.json() for user in users_list if user.is_admin]
+
+        return list_of_users, 200
```

Comment:
if you are returning on admin.py resource the 200, why do you need to send it here?
You don't do anything with this result there https://github.com/anitab-org/mentorship-backend/pull/459/files#diff-40b144b4232d9ee0bc35b84eb4226c7cR107

Reply:
oops! i have fixed the code now

Reply:
Sorry! I have fixed it now